GitOut/Features/Settings/SettingsPage.xaml

```diff
@@ -170,14 +156,103 @@
                             <StackPanel>
                                 <TextBlock Text="Theme" Style="{StaticResource Header6TextStyle}" />
                                 <themes:ThemeSettingsPicker
-                                    SelectThemeCommand="{Binding SelectThemeCommand}"
-                                    ThemeSelected="{Binding DataContext.ChangeThemeCommand, RelativeSource={RelativeSource AncestorType={x:Type local:SettingsPage}}}"
+                                    SelectThemeCommand="{Binding ChangeThemeCommand}"
                                 />
                             </StackPanel>
                         </Border>
                     </StackPanel>
                 </Grid>
             </ScrollViewer>
+        </DataTemplate>
+        <DataTemplate DataType="{x:Type local:ProcessSettingsViewModel}">
+            <ListView
+                Style="{StaticResource MaterialListStyle}"
+                SelectionMode="Single"
+                ItemsSource="{Binding Reports}"
+            >
+                <ListView.ItemTemplate>
+                    <DataTemplate DataType="{x:Type process:ProcessEventArgs}">
+                        <Grid>
+                            <Grid>
+                                <Grid.ColumnDefinitions>
+                                    <ColumnDefinition Width="*"/>
+                                    <ColumnDefinition Width="Auto"/>
+                                </Grid.ColumnDefinitions>
+                                <TextBlock Style="{StaticResource MonoSpacedTextStyle}">
+                                    <Run Text="{Binding ProcessName, Mode=OneWay}" Foreground="Yellow"/>
+                                    <Run Text="{Binding Options.Arguments, Mode=OneWay}"/>
+                                    <LineBreak />
+                                    <Run Text="{Binding WorkingDirectory.Directory, Mode=OneWay}"/>
+                                </TextBlock>
+                                <TextBlock Grid.Column="1" Style="{StaticResource NormalTextStyle}">
+                                    <Run Text="{Binding StartTime, StringFormat={}{0:H:mm:ss}, Mode=OneWay}"/>
+                                    <Run Text="{Binding Duration, StringFormat={}{0:s\\.fff}s, Mode=OneWay}"/>
+                                </TextBlock>
+                            </Grid>
+
+                            <Popup
```

Comment:
maybe we can use a different itemtemplate if an item is selected, which shows the output in a scrollable contentpresenter instead. if the output is many lines it gets truncated by the window
![image](https://user-images.githubusercontent.com/38684932/110388574-8e277400-8063-11eb-943c-39f3c3118b9b.png)

Reply:
Yes, nice suggestion, will change :+1: